docs/osf-trivia-list/dts.md

```diff
@@ -17,6 +17,25 @@ buying a [Dasharo Entry Subscription](../ways-you-can-help-us.md#become-a-dashar
 Commercial use of DTS should be discussed directly with
 [3mdeb](mailto:leads@3mdeb.com) or [Dasharo Team](mailto:contact@dasharo.com).
 
+### How can I use my Dasharo Entry Subscription credentials
+
+<!-- Need to be replaced in case the menu changed. -->
+
+After purchasing the Dasharo Entry Subscription, you should receive an email
+with keys to use with [Dasharo Tools Suite](../dasharo-tools-suite/overview.md).
+This section describes how to do it. As also mentioned in the email, the
+easiest way is to run DTS from a USB flash drive. Documentation on this is
+included [here](../dasharo-tools-suite/documentation.md#bootable-usb-stick).
+After booting, you will see a text menu. Of interest here is option number 4,
+`Load your DES keys`. After selecting it, rewrite the credentials received in
+the following order: `logs key`, `firmware download key`, and `password`. After
+entering these credentials, they will be verified by attempting to connect to
+our server. If successful, the message `Verification of the Dasharo DES was
+successful. They are valid and will be used.` will be displayed. Below is a
+short video that present loading DES keys.
+
+![IMG](img/des-creds.gif)
```

Comment:
Just to confirm, those credentials aren't working, right? We should revoke them, so nobody can use them from this demo.

Reply:
Yes, the subscription with those creds is cancelled.